phpBB/phpbb/message/message.php

```diff
@@ -258,15 +258,19 @@ public function send(\phpbb\di\service_collection $messenger, $contact)
 					$messenger_method->set_addresses($recipient);
 					$messenger_method->reply_to($this->sender_address);
 
-					$messenger_method->header('X-AntiAbuse', 'Board servername - ' . $this->server_name);
```

Comment:
Why not add a header() or addHeader() method to the interface and then implement this as empty method for non email methods?